Content.Server/Power/Components/CablePlacerComponent.cs

```diff
@@ -1,16 +1,38 @@
 using Robust.Shared.Prototypes;
 using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
 using Content.Shared.Power;
+using Content.Shared.Whitelist;
 
 namespace Content.Server.Power.Components
 {
     [RegisterComponent]
     public sealed partial class CablePlacerComponent : Component
     {
+        /// <summary>
+        /// The structure prototype for the cable coil to place.
+        /// </summary>
         [DataField("cablePrototypeID", customTypeSerializer:typeof(PrototypeIdSerializer<EntityPrototype>))]
         public string? CablePrototypeId = "CableHV";
 
-        [DataField("blockingWireType")]
+        /// <summary>
+        /// What kind of wire prevents placing this wire over it as CableType.
+        /// </summary>
+        [DataField]
         public CableType BlockingCableType = CableType.HighVoltage;
+
+        /// <summary>
+        /// Block placement over entities by blacklist instead of CableType?
+        /// </summary>
+        [DataField]
+        public bool UsePlaceOverBlacklist = false;
+
+        [DataField]
+        public EntityWhitelist Blacklist { get; private set; } = new();
+
```

Comment:
Having duplicate fields that may or may not be toggled on is bad to work with, just make it use the blacklist entirely instead.